pandas/tests/frame/test_dtypes.py

```diff
@@ -669,6 +669,19 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    from operator import (add, mul, floordiv, sub)
```

Comment:
I see no reason why we shouldn't move this import to the top.  You *might* even consider just doing "import operator as ops" so that you can namespace with (`ops.[operation-name]`).